images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -11,38 +11,78 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     PYTHONIOENCODING="UTF-8" \
     LANG="C.UTF-8" \
     LANGUAGE="C.UTF-8" \
-    LC_ALL="C.UTF-8"
-
-ARG IMAGES_DIR="/opt/tmp/images"
-RUN oryx prep --skip-detection --platforms-and-versions nodejs=14 --debug \
-    && echo "$DEBIAN_FLAVOR" \
```

Comment:
Removed oryx prep step as I believe it installs nodejs, yarn and python dependencies redundantly.